libs/common/setup.go

```diff
@@ -58,6 +60,16 @@ func Setup(serviceName, version string, auth bool) {
 
 		setupAuth()
 	}
+
+	organizationIdStr := hwutil.GetEnvOr("ORGANIZATION_ID", "")
```

Comment:
rename these variables consistent with the global variable (I suggested it above somewhere)

Reply:
maybe add a comment why this variable exists and when it is used